docs/operations/how-to-clone-a-volume.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "How To Clone a Volume"
+linkTitle: How To Clone a Volume
+---
+
+## Create a Clone of an Existing Volume
+
+The Kubernetes [Volume Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) feature provides users with a mechanism to create a clone Volume by adding a dataSource that references an existing Volume.
```

Comment:
No need to explain it again here, it's already done on the /docs/concepts/cloning.md
We can add a link to that page

~The Kubernetes [Volume Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) feature provides users with a mechanism to create a clone Volume by adding a dataSource that references an existing Volume.~

[Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) a volume is done by referring to an existing one in the "dataSource" field of the manifest.